react/components/EXPERIMENTAL_Table/Sections/index.tsx

```diff
@@ -34,7 +34,8 @@ function Sections(
     <div
       style={{ height: disableScroll ? 'auto' : tableHeight, ...motion }}
       className={classNames('mw-100', ORDER_CLASSNAMES.TABLE, {
-        'overflow-x-auto overflow-y-auto overflow-hidden': !disableScroll,
+        'overflow-x-auto overflow-y-auto': !disableScroll,
+        'overflow-x-auto': disableScroll,
```

Comment:
Add `overflow-x-auto` to `classNames`. What do you think?

```suggestion
        'overflow-y-auto': !disableScroll,
```